### PR TITLE
Added DataTableGroupBy

### PR DESCRIPTION
**Pull request by ericsoderberghp**

### CODE REVIEW

File: src/screens/DataTableGroupBy.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  Description,
+  Example,
+} from '../components/Doc';
+
+const DataTableGroupByPage = () => (
+  <Page>
+    <ComponentDoc
+      name="DataTableGroupBy"
+      stable
+      availableAt={[
```
Comment (jcfilben) [inline src/screens/DataTableGroupBy.js:19]: Can we add a storybook link here? https://storybook.grommet.io/?path=/story/data-datatablegroupby-simple--simple

**Review by jcfilben — COMMENTED** [review]
Looks like `yarn.lock` needs updated again

File: src/screens/DataTableGroupBy.js (hunk 1 above)
Comment (ericsoderberghp) [reply src/screens/DataTableGroupBy.js:19]: good catch, done

**Review by jcfilben — APPROVED** [review]
Looks good, just need to update yarn.lock

**Comment by ericsoderberghp** [comment]
> Looks good, just need to update yarn.lock

I think the issue is that the NODE_VERSION in Netlify needs to be updated. Done, rerunning ...

**Pull request merged**